doc/src/manual/faq.md

```diff
@@ -22,11 +22,28 @@ On the other hand, language *interoperability* is extremely useful: we want to e
 
 ### How does Julia define its public API?
 
-Julia `Base` and standard library functionality described in the
-[the documentation](https://docs.julialang.org/) that is not marked as unstable
-(e.g. experimental and internal) is covered by [SemVer](https://semver.org/).
-Functions, types, and constants are not part of the public API if they are not
-included in the documentation, _even if they have docstrings_.
+Julia's public [API](https://en.wikipedia.org/wiki/API) is the behavior described in
+documentation of public symbols from `Base` and the standard libraries. Functions,
+types, and constants are not part of the public API if they are not public, even if
+they have docstrings or are described in the documentation. Further, only the documented
+behavior of public symbols is part of the public API. Undocumented behavior of public
+symbols is internal.
```

Comment:
There are still some symbols which are documented in the manual and not yet marked as public (e.g. `Base.EnvDict`). Is there a plan to try to find them all? otherwise should we also keep around for the time being the old version mentioning the documentation?

Reply:
I sorta responded to this way up the thread: https://github.com/JuliaLang/julia/pull/50105#issuecomment-1664769908

The plan is to find them all by 1.11. https://github.com/JuliaLang/julia/issues/51335.